wolfcrypt/src/integer.c

```diff
@@ -4223,6 +4223,10 @@ static int mp_div_d (mp_int * a, mp_digit b, mp_int * c, mp_digit * d)
       q.used = a->used;
       q.sign = a->sign;
   }
+  else {
+      mp_init(&q); /* initialize to help static analysis */
```

Comment:
mp_init in integer.c has no XMALLOCs and fp_init calls fp_zero which both have no XMALLOCs.

Will run some more testing with valgrind and memory checking tools to verify.